base/uk.ac.stfc.isis.ibex.ui.configserver/src/uk/ac/stfc/isis/ibex/ui/configserver/commands/NewBlockHandler.java

```diff
@@ -47,13 +47,16 @@
 import uk.ac.stfc.isis.ibex.ui.configserver.ConfigurationViewModels;
 import uk.ac.stfc.isis.ibex.ui.configserver.editing.blocks.EditBlockDialog;
 
+import org.apache.logging.log4j.Logger;
+import uk.ac.stfc.isis.ibex.logger.IsisLog;
+
 /**
  * The handler class for creating a new block with a default PV.
  */
 public class NewBlockHandler extends AbstractHandler {
     static final ConfigServer SERVER = Configurations.getInstance().server();
     static final Shell SHELL = PlatformUI.getWorkbench().getActiveWorkbenchWindow().getShell();
-    
+    private static final Logger LOG = IsisLog.getLogger(NewBlockHandler.class);
```

Comment:
You have not yet tidied up the redundant code changes here. Remove `LOG` (it is never used) and the imports of Logger and IsisLog